src/borg/repository.py

```diff
@@ -1084,6 +1093,9 @@ class LoggedIO:
     class SegmentFull(Exception):
         """raised when a segment is full, before opening next"""
 
+    class AtticSegment(Error):
+        """Attic segment detected [segment {}]"""
```

Comment:
Huh? I am pretty sure you cannot do this in a doc string... Do you mean `"""Attic segment detected [segment {}, offset 0]"""` instead?